en/flying/pre_flight_checks.md

```diff
@@ -40,10 +41,39 @@ The following errors (with associated checks and parameters) are reported by the
 
 #### PREFLIGHT FAIL: EKF HIGH IMU ACCEL BIAS
 
-* This error is produced when the IMU accelerometer bias estimated by the EKF is excessive. 
-* Excessive in this case means that the bias estimate exceeds half the configured limit.
-  The limit is defined in the [EKF2_ABL_LIM](../advanced_config/parameter_reference.md#EKF2_ABL_LIM) parameter.
- 
+<!-- https://github.com/PX4/PX4-Autopilot/blob/master/src/modules/commander/Arming/PreFlightCheck/checks/ekf2Check.cpp#L267 -->
+<!-- Useful primer on biases: https://www.vectornav.com/resources/inertial-navigation-primer/specifications--and--error-budgets/specs-imuspecs -->
+<!-- Mathieu Bresciani is expert -->
+
+The EKF IMU acceleration bias is the difference between the measured acceleration reported by the IMU sensor and the expected acceleration reported by the EKF2 estimator (which fuses position and/or velocity data from a number of sources, including the IMU, GNSS, flow sensors etc.).
+This bias may change when the sensor is turned on (“turn-on bias”) and over time due to noise and temperature differences (“in-run bias”).
+The number should generally be very small (near zero), indicating that measurements from different sources all agree on the acceleration.
+
+The warning indicates that the bias is higher than some arbitrary threshold.
+It is most likely a sign that accelerometer or thermal calibration are required:
+
+- If you _sometimes_ get the warning: [re-calibrate the accelerometer](../config/accelerometer.md).
+- If you get _regularly_ get the warning: Perform a [thermal calibration](../advanced_config/sensor_thermal_calibration.md).
+- If you still get the warning after thermal calibration (or you can't perform thermal calibration): 
+  - Verify that the issues does not come from the sensor, or autopilot hardware
```

Comment:
@bresch How might you verify this? If we give an instruction, we should provide hints or link to next steps.

Reply:
The "simpler" check would be to try with another autopilot for example. Otherwise do some bench test logging runs and compare the output of all the accelerometers. (To log all the accelerometers, enable "sensor comparison" in [SDLOG_PROFILE](https://docs.px4.io/master/en/advanced_config/parameter_reference.html#SDLOG_PROFILE))